Doppler.HtmlEditorApi/Storage.DapperProvider/DapperCampaignContentRepository.cs

```diff
@@ -136,4 +136,15 @@ public async Task SaveNewFieldIds(int ContentId, IEnumerable<int> fieldsId)
 
         await _dbContext.ExecuteAsync(new SaveNewCampaignFields(ContentId, fieldsId));
     }
+
+    public async Task SaveLinks(int ContentId, IEnumerable<string> links)
+    {
+        if (links.Any())
+        {
+            // TODO: insert new links
+        }
+        // TODO: Delete removed links from AutomationConditional table
+        // See https://github.com/MakingSense/Doppler/blob/ed24e901c990b7fb2eaeaed557c62c1adfa80215/Doppler.Application.CampaignsModule/Services/Classes/CampaignContentService.cs#L2519-L2523
+        await _dbContext.ExecuteAsync(new DeleteRemovedCampaignLinks(ContentId, links));
```

Comment:
@MarianoHerrera 

> I'm asuming this delete will be executed before insertion to work.

The delete is being done after insert, but I do not see a problem there. What am I missing?

Reply:
Sorry, my bad..... It was just to see if you were paying attention 
![giphy 1](https://user-images.githubusercontent.com/6794856/160459127-2291976c-3d7e-4638-87dc-423b40445c15.gif)